scripts/apis_scan.py

```diff
@@ -298,7 +299,14 @@ def generate_report(scan_timestamp):
 
 if __name__ == "__main__":
 
-    workDir = resultDir + sys.argv[1] + "/"
+    parser = argparse.ArgumentParser(description='Connect to ZAP and launch a scan based on config.yaml')
+    parser.add_argument('--debug', action='count', default=0, help='Show debugging output')
+    parser.add_argument('destination', metavar='N',
+                                help=f"Directory of the report, relative to {resultDir}")
+
+    logging.basicConfig(level=logging.DEBUG if parser.parse_args().debug else logging.INFO)
+
+    workDir = resultDir + parser.parse_args().destination + "/"
```

Comment:
I don't think that it's a good choice to use concatenation like this.
Consider a case where some of the variables is not a string. In this case you'll get an error:
```bash
>>> "str" + 1
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate str (not "int") to str
```

so it's much safer to use a `%` formatter:
```
>>> s = "str"
>>> n = 1
>>> "{}{}".format(s, n)
'str1'
```

or f-strings:
```
>>> s = "str"
>>> n = 1
>>> f"{s} {n}"
'str 1'
```

Do you think it's better to replace the line with f-string formatter?

Reply:
not my change : I only needed to change the name. But if we're ok to not have python-2 compatibility, I'm all for using f-strings